packages/exhibitor-migration-script/README.md

```diff
@@ -0,0 +1,48 @@
+exhibitor-migration-script
+==========================
+
+When a user is updating to DC/OS 1.9 from an older version we need to update several values in exhibitor to use the new 
+filesystem paths for zookeeper state on disk.
+
+This package includes two scripts to help with managing the update.
+
+
+## `dcos-exhibitor-migrate-status`
+
+Will query exhibitor (at `localhost`) and check if the migration has been completed, in progress, or has not yet been
```

Comment:
127.0.0.1 rather than localhost. Hostnames are weird / off / localhost not always resolvable in some environments we've run into.